app-timer-run/src/main/res/layout-land/activity_screen.xml

```diff
@@ -9,84 +9,67 @@
 
     <TextView
         android:id="@+id/textStepInfo"
-        android:layout_width="0dp"
-        android:layout_height="0dp"
-        android:layout_marginStart="8dp"
-        android:layout_marginEnd="8dp"
+        android:layout_width="191dp"
+        android:layout_height="113dp"
+        android:layout_marginTop="16dp"
```

Comment:
Designing layouts with the layout inspector makes the layout inconsistent on different devices, which is why many developers prefer XML.

Reply:
Ah, I didn't know using the layout designer was known to be a problem. Will remember this next time I work on native Android project